docs/apidiffs/1.7.1_vs_1.7.0/opentelemetry-sdk.txt

```diff
@@ -0,0 +1,2 @@
+Comparing source compatibility of  against 
```

Comment:
It looks weird that the versions are empty here. Is that normal?

Reply:
yeah.. this is a "feature" of japicmp, unfortunately. I have tried digging to fix it, but it ended up being more trouble than it's worth.